tools/internal_ci/macos/grpc_distribtests_php.sh

```diff
@@ -39,7 +42,7 @@ cp -r artifacts/php_pecl_package_macos_*/* input_artifacts/ || true
 # a better signal about which distribtest are affected by the currently broken artifact builds.
 tools/run_tests/task_runner.py -f distribtest macos php -j 4 -x distribtests/sponge_log.xml || FAILED="true"
 
-tools/internal_ci/helper_scripts/delete_nonartifacts.sh || true
+tools/internal_ci/helper_scripts/store_artifacts_from_moved_src_tree.sh
```

Comment:
This is a distribtest, do we need to preserve artifacts?

Reply:
see comment above.